### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**

### CODE REVIEW

File: lib/shellhandlerapt.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def _check_versions(self, package, update=True):
             if installed == "(none)":
                 installed = False
             if candidate == "(none)":
-                candiate = False
+                candidate = False
```
Comment (jsoref) [inline lib/shellhandlerapt.py]: This is a code change. The old code would probably have done effectively nothing...

File: lib/aptdaemonhandler.py [hunk 2]
```diff
@@ -26,7 +26,7 @@
 except:
     log('python apt import error')
 
-class AptdeamonHandler:
+class AptdaemonHandler:
```
Comment (jsoref) [inline lib/aptdaemonhandler.py]: This is an api change (along with renaming the file)

**Comment by garbear** [comment]
You are doing tedious yet noble work. Thank you.

**Comment by Razzeee** [comment]
Can you rebase?

**Comment by Razzeee** [comment]
This also misses a typo, see https://github.com/arpitn30/xbmc/pull/12#discussion-diff-133174442R125

**Comment by jsoref** [comment]
In general, I can rebase, but at this moment I don't have a computer handy.

**Comment by MartijnKaijser** [comment]
Already picked up by https://github.com/XBMC-Addons/service.xbmc.versioncheck/pull/12

**Comment by Razzeee** [comment]
Thank you, I hope your not offended by me rebasing :)

**Comment by jsoref** [comment]
Not at all.

**Pull request closed without merging**